charts/aws-vpc-cni/templates/_helpers.tpl

```diff
@@ -55,3 +55,25 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+The aws-vpc-cni-init image to use
+*/}}
+{{- define "aws-vpc-cni.initImage" -}}
+{{- if .Values.init.image.override }}
+{{- .Values.init.image.override }}
+{{- else }}
+{{- printf "%s.dkr.%s.%s.%s/amazon-k8s-cni:%s" .Values.init.image.account .Values.init.image.endpoint .Values.init.image.region .Values.init.image.domain .Values.init.image.tag }}
+{{- end }}
+{{- end }}
+
+{{/*
+The aws-vpc-cni image to use
+*/}}
+{{- define "aws-vpc-cni.image" -}}
+{{- if .Values.image.override }}
+{{- .Values.image.override }}
+{{- else }}
+{{- printf "%s.dkr.%s.%s.%s/amazon-k8s-cni:%s" .Values.image.account .Values.image.endpoint .Values.image.region .Values.image.domain .Values.image.tag }}
+{{- end }}
+{{- end }}
```

Comment:
If it works, that's fine, I was just trying to stay consistent with the other uses in this file, but it looks like there is a mix. I think we want the `-` for the trailing end for define, as it would delete whitespace if someone passed it in the `tag`, but other than that, it's fine

Reply:
> I think we want the - for the trailing end for define, as it would delete whitespace if someone passed it in the tag, but other than that, it's fine

I'm not sure I follow? It works correct as it is currently.

Reply:
Sure, and it would continue to work fine if you put a trailing dash on this line and on line 68

Reply:
Alright, let's not spend any more time on this. Dropping